src/app/features/asset-list/components/stacks-balance-list-item.layout.stories.tsx

```diff
@@ -0,0 +1,64 @@
+import { TooltipProvider } from '@radix-ui/react-tooltip';
+import { Meta, StoryObj } from '@storybook/react';
+import BigNumber from 'bignumber.js';
+
+import { StacksBalanceListItemLayout } from './stacks-balance-list-item.layout';
+
+const meta: Meta<typeof StacksBalanceListItemLayout> = {
+  component: StacksBalanceListItemLayout,
+  tags: ['autodocs'],
+  title: 'Feature/StacksBalanceListItem',
```

Comment:
We should make sure to coordinate with @pete-watters on the new sections being added to our storybook. I noticed in his he has new sections too.

Reply:
I actually decided to revert my changes to fit into the current system. Once everything is merged lets review it together. 

I had just added `Design System/` to the start of the path